diagnostic.md

```diff
@@ -57,13 +70,13 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('li').prev('body')
```

Comment:
This will attempt to select a preceding sibling element of `<li>` called `<body>`, which doesn't exist since `<body>` is an indirect ancestor of `<li>`.